parser/parser.go

```diff
@@ -64,12 +64,67 @@ type Node interface {
 }
 
 type AST struct {
-	Select  *Select `(   "SELECT"  @@  `
-	Insert  *Insert `  | "INSERT"  @@  `
-	Replace *Insert `  | "REPLACE" @@ )`
-	End     string  `( ";" )?`
+	Select      *Select      `(   "SELECT"         @@`
+	Insert      *Insert      `  | "INSERT"         @@`
+	Replace     *Insert      `  | "REPLACE"        @@`
+	CreateTable *CreateTable `  | "CREATE" "TABLE" @@ ) ";"?`
 }
 
+type CreateTable struct {
+	Table   string              `@(Ident | QuotedIdent) "("`
```

Comment:
This should be `( @Ident ( @"." @Ident )* | @QuotedIdent )`. Dots are valid in dynamodb table names

Reply:
Aha!